app/assets/stylesheets/application.css

```diff
@@ -71,6 +72,8 @@ th h3, th p {
     background-color: #fff;
 }
 
+about.countainer
```

Comment:
We could do it without creating a whole new layout. You have the code for the about page in the layout instead of using a yield to go back to the /welcome/about.html.erb.

There's an easier way to do that then creating a whole new template just for one page. If you like, I can help you with it?